manifests/eap/client.pp

```diff
@@ -54,6 +59,8 @@
 #        auth_cert_source     => 'puppet:///path/to/your/lb-myapp.pkcs12',
 #        auth_cert_password   => hiera('myapp::lightblue::pass'),
 #        ssl_ca_source        => 'puppet:///path/to/your/cacert.pem',
+#        ssl_ca_file_path     => 'cacert.pem',
+#        ca_certificates      => {'cacert.pem' => {'source' => 'puppet:///path/to/your/cacert.pem', 'file' => 'cacert.pem'}}
```

Comment:
No if you specify both, the multiple will override the single.  I updated the comments and example section to reflect it is one or the other